jobs/config.json

```diff
@@ -395,6 +395,28 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-1-6-1-7-master-skew": {
```

Comment:
if it's a kubeadm job, add kubeadm to your job name, or else it's hard to tell the difference between this and a regular gce upgrade-skew job :smiling_imp:

Reply:
All gce upgrade-skew tests have upgrade in the name (and not skew). I feel adding kubeadm won't be required.

Reply:
I think all kubeadm jobs will need to chain to a bazel-build job in order to get the right k8s version? cc @pipejakob here

Reply:
kubeadm should be in the name, and it should be chained to the bazel job